scenarioo-client/app/manage/generalSettings/generalSettings.controller.js

```diff
@@ -43,6 +44,10 @@ function GeneralSettingsController($scope, BranchesResource, ConfigService, Vers
             }
         );
 
+        DiffViewerConfigResource.get(function onSuccess(result) {
```

Comment:
I propose to refactor DiffViewerConfigResource and put it together with SearchEngineStatusService and have instead one endpoint to get all these statuses at once. We could simply call it ScenariooStatusResource (also the REST-endpoint should become the same). @danielsuter waht you think? I understand the argument, that it belongs not to the config endpoint.